docs/src/maintain/manage-releases.md

```diff
@@ -15,7 +15,11 @@ Releases are when a project formally publishes a new version so the community ca
 
 ## Release Team
 
-A two-person release team is assigned to each scheduled release. This two-person team is responsible for:
+A two-person release team is assigned to each scheduled release.
+
+TODO: add a sentence or two about who the release team is composed of...just TSC members? how is the release team determined?
```

Comment:
to reviewer: could you answer the question in the TODO?

Reply:
Yes, just TSC members. Actually, one TSC member as we no longer have two-person teams for releases, just one person. Determined in the TSC meeting day before the release.

Reply:
udpated to refere to 'release manager' instead of 'release team' throughout PR